blocks/components/editable/index.js

```diff
@@ -127,6 +146,22 @@ export default class Editable extends wp.element.Component {
 		}
 	}
 
+	componentWillReceiveProps( nextProps ) {
+		forEach( nextProps.formats, ( state, format ) => {
+			const currentState = this.formats[ format ] || false;
```

Comment:
I would prefer if we find a way to avoid the cache `this.formats` but it's ok for now.